### PR TITLE
Fix Compare View Predominantly Forested Chart Values

### PR DESCRIPTION
**Pull request by arottersman**
## Overview

- Extract helper functions out to handle getting tr-55 results values
- Use the helper functions in the compare charts and now tables, regular tr-55 charts and tables
- Cleanup `tr55/???/views` — still had a a bit of code referencing the old compare view

Connects #2379

### Demo

screen shot 2017-10-18 at 5 43 13 pm
screen shot 2017-10-18 at 5 43 07 pm

### Notes
`utils` probably isn't the greatest forever home for these helper functions. Someday we may wish to refactor the `ResultModel` or maybe the `ScenarioModel` so it has proper TR-55 and Mapshed subclasses, and then place the helpers on the TR-55 subclass.

## Testing Instructions

 * Pull and `bundle`
 * Create a tr-55 project for an easily findable huc. Give it a few scenarios, and cover one in forest landcover
 * Do the same for the same huc on staging
 * Compare the displayed results between staging and this branch and confirm they're the same
 * Open the compare view on this branch and confirm the chart and table values match when rounding has been accounted for

### CODE REVIEW

**Comment by kellyi** [comment]
Taking a look at this.

File: src/mmw/js/src/core/utils.js [hunk 1]
```diff
@@ -416,6 +416,27 @@ var utils = {
         return 'modified';
     },
 
+    /** Access a scenario's result for a given type
+     *   @param {ScnenarioModel} scenario
```
Comment (rajadain) [inline src/mmw/js/src/core/utils.js]: Typo `ScnenarioModel` → `ScenarioModel`

**Comment by kellyi** [comment]
Values now differ. Here's the same AOI (and same modifications) on staging:

screen shot 2017-10-19 at 11 36 29 am

... and on this branch ...

screen shot 2017-10-19 at 11 36 36 am

**Comment by kellyi** [comment]
> If something can be surreptitiously slipped in here, I'd like the order of the rows in the table to be changed to match those of the charts.

In favor of handling this in another card.

**Comment by rajadain** [comment]
Actually the Modeling View makes the same mistake (of value ordering):

image

so that should be its own card. I'll make one.

**Pull request merged**